### PR TITLE
add cookiefactoryv3

### PR DESCRIPTION
**Pull request by johnnyw-aws**
*Description of changes:* add latest v3 version of cookiefactory demo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by johnnyw-aws** [comment]
tested install steps + running app locally on OSX against both us-east-1 and us-west-2

File: src/workspaces/cookiefactoryv3/readme.assistant.md
Comment (jiajzhou) [inline src/workspaces/cookiefactoryv3/readme.assistant.md:1]: I think we should move this under cookiefactoryv3/assistant folder

File: src/workspaces/cookiefactoryv3/dashboard/src/config/cognito.template.ts
Comment (jiajzhou) [inline src/workspaces/cookiefactoryv3/dashboard/src/config/cognito.template.ts:1]: MINOR: I don't think this file is needed. All the config is in the app.config.tsx file now.

File: src/workspaces/cookiefactoryv3/dashboard/src/config/sites.template.ts
Comment (jiajzhou) [inline src/workspaces/cookiefactoryv3/dashboard/src/config/sites.template.ts:1]: MINOR: I don't think this file is needed.

File: src/workspaces/cookiefactoryv3/dashboard/src/config/users.template.ts
Comment (jiajzhou) [inline src/workspaces/cookiefactoryv3/dashboard/src/config/users.template.ts:1]: MINOR: I don't think this file is needed.

Comment (johnnyw-aws) [reply src/workspaces/cookiefactoryv3/dashboard/src/config/users.template.ts:1]: updated in e20d2792

File: src/workspaces/cookiefactoryv3/dashboard/src/config/sites.template.ts
Comment (johnnyw-aws) [reply src/workspaces/cookiefactoryv3/dashboard/src/config/sites.template.ts:1]: updated in e20d2792

File: src/workspaces/cookiefactoryv3/dashboard/src/config/cognito.template.ts
Comment (johnnyw-aws) [reply src/workspaces/cookiefactoryv3/dashboard/src/config/cognito.template.ts:1]: updated in e20d2792

File: src/workspaces/cookiefactoryv3/readme.assistant.md
Comment (johnnyw-aws) [reply src/workspaces/cookiefactoryv3/readme.assistant.md:1]: thanks, sounds good, updated in e20d2792

**Review by jiajzhou — APPROVED** [review]
LGTM

**Pull request merged**